libs/components/core/src/lib/modules/log/log.service.spec.ts

```diff
@@ -1,25 +1,465 @@
-import { TestBed, inject } from '@angular/core/testing';
+import { TestBed } from '@angular/core/testing';
 
 import { SkyLogService } from './log.service';
+import { SkyLogLevel } from './types/log-level';
+import { SKY_LOG_LEVEL } from './types/log-level-token';
 
 describe('Log service', () => {
+  let consoleDebugSpy: jasmine.Spy;
+  let consoleErrorSpy: jasmine.Spy;
+  let consoleInfoSpy: jasmine.Spy;
+  let consoleLogSpy: jasmine.Spy;
+  let consoleWarnSpy: jasmine.Spy;
+  let logService: SkyLogService;
+
+  function setupSpys(): void {
```

Comment:
`setupSpies` 😄

Reply:
I would fix this.... but I changed it out for the spy object - so this is now gone